studio/keyvalue_provider.py

```diff
@@ -425,11 +424,12 @@ def register_user(self, userid, email):
         if existing_email != email:
             self._set(keypath, email)
 
+    def get_artifact_store(self):
+        return self.store
+
```

Comment:
Convenience getter.

Reply:
Do these help with things like making sure someone cannot modify the value inside the main python object ?

Reply:
Well, that's the general intent, but in reality, Python doesn't provide any access control on language level, so all you can do if use special naming conventions as a hint to whoever is reading the code.